components/Cards/Card.js

```diff
@@ -0,0 +1,47 @@
+import React from "react";
```

Comment:
Can you change Card.js file to something more specific too? I'm not sure what you're referring to. Is it a general Card? If so, what's wrong with the one from MaterialUI?